docs/sources/tempo/configuration/_index.md

```diff
@@ -1204,6 +1232,99 @@ memberlist:
 
 ```
 
+## Configuration Blocks
```

Comment:
Thank you for adding this section of doc. We use sentence case in our docs. For example, the headings should be like the one below instead of title case. For more information, refer to the [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/write/style-guide/capitalization-punctuation/).
```suggestion
## Configuration blocks
```